doc/source/install/frontend-install.rst

```diff
@@ -2,77 +2,81 @@
 Installation du frontend
 ========================
 
-Vous voulez nous aider au développement du frontend ? Installez Node.js et npm grâce aux instructions qui suivent !
+Vous voulez nous aider au développement du frontend ? Installez Node.js et Yarn grâce aux instructions qui suivent !
 
 
-Installation de Node.js et npm
+Installation de Node.js et Yarn
 ==============================
 
 Le frontend de Zeste de Savoir repose sur la version supportée à long terme actuelle de Node.js, la version 4.
 
-L'installation de ``node`` est détaillée sur le site de Node.js, soit par `téléchargement direct <https://nodejs.org/en/download/>`_, soit via un `gestionnaire de paquet <https://nodejs.org/en/download/package-manager/>`_. Zeste de Savoir requiert la version 4, mais marche également avec la version 6.
+L’installation de Node.js avec [nvm](https://github.com/creationix/nvm) est relativement simple. De plus, nvm permet très facilement de basculer d’une version à une autre.
+
+Autrement, l'installation de ``node`` est détaillée sur le site de Node.js, soit par `téléchargement direct <https://nodejs.org/en/download/>`_, soit via un `gestionnaire de paquet <https://nodejs.org/en/download/package-manager/>`_. Zeste de Savoir requiert la version 4, mais marche également avec la version 6.
+
 
 Vérifier que les bonnes versions sont installées
 ------------------------------------------------
 
-Pour vérifier que Node.js et npm sont installés (et que vous avez les bonnes versions) :
+Pour vérifier que Node.js et yarn sont installés (et que vous avez les bonnes versions) :
 
 .. sourcecode:: bash
 
-    node -v
+    $ node -v
     v4.x.x
-    npm -v
-    2.14.12 # n'importe quelle version plus grande que 1.x
+    $ yarn -v
+    0.27.x
+
+Si `yarn` n’est pas installé, vous pouvez le rajouter avec `npm i -g yarn`.
 
-**Vous devez avoir une version de Node.js v4.x et de npm 2 ou supérieure.** Si votre version de npm est 1.x.x, vous devez le mettre à jour (voir juste en dessous).
+Vous devez avoir au minimum la version 4.x.x de Node.js.
 
-Mise à jour de Node.js et npm
+Mise à jour de Node.js et Yarn
 -----------------------------
 
-Pour npm, il suffit de le mettre à jour avec cette commande :
+Pour Yarn, il suffit de le mettre à jour avec cette commande :
 
 .. sourcecode:: bash
 
-    npm install -g npm # si ça échoue, retentez avec sudo
+    $ yarn install -g yarn
 
-Pour ce qui est de Node.js, une mise à jour via le gestionnaire de paquets devrait fonctionner.
-
-.. note::
-   En cas de gros pépin avec l'installation de npm, il peut arriver que ce dernier soit complètement en vrac. Dans ce genre de situation, ce `lien StackOverflow <http://stackoverflow.com/questions/31025048/npm-doesnt-work-get-always-this-error-error-cannot-find-module-are-we-ther>`_ peut vous sauver ;) .
+Pour ce qui est de Node.js, utilisez nvm si possible.
```

Comment:
Ouais je comprends les réserves, mais là le markdown c'est un requirement dur, y'a pas le choix.

Hésite pas à mettre un lien vers <https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions> s'il le faut. 6 est la LTS actuelle, dans 2 mois 8 sera LTS, c'est pas déconnant d'exiger 6 actuellement.